openpype/hosts/unreal/plugins/create/create_look.py

```diff
@@ -1,56 +1,55 @@
 # -*- coding: utf-8 -*-
-"""Create look in Unreal."""
-import unreal  # noqa
-from openpype.hosts.unreal.api import pipeline, plugin
-from openpype.pipeline import LegacyCreator
+import unreal
 
+from openpype.hosts.unreal.api.pipeline import (
+    create_folder
+)
+from openpype.hosts.unreal.api.plugin import (
+    UnrealAssetCreator
+)
 
-class CreateLook(LegacyCreator):
+
+class CreateLook(UnrealAssetCreator):
     """Shader connections defining shape look."""
 
-    name = "unrealLook"
-    label = "Unreal - Look"
+    identifier = "io.openpype.creators.unreal.look"
+    label = "Look"
     family = "look"
     icon = "paint-brush"
 
-    root = "/Game/Avalon/Assets"
-    suffix = "_INS"
-
-    def __init__(self, *args, **kwargs):
-        super(CreateLook, self).__init__(*args, **kwargs)
-
-    def process(self):
-        name = self.data["subset"]
-
+    def create(self, subset_name, instance_data, pre_create_data):
         selection = []
-        if (self.options or {}).get("useSelection"):
+        if pre_create_data.get("use_selection"):
```

Comment:
So there is not an option to use this creator without selection? In that case the `"use_selection"` should not be in pre-create attributes. If it's disabled it crashes so it's pointless to have that attribute there.